app/src/preview/java/org/mozilla/focus/utils/AdjustHelper.java

```diff
@@ -5,10 +5,78 @@
 
 package org.mozilla.focus.utils;
 
-import android.content.Context;
+import android.app.Activity;
+import android.app.Application;
+import android.os.Bundle;
+import android.text.TextUtils;
+
+import com.adjust.sdk.Adjust;
+import com.adjust.sdk.AdjustConfig;
+import com.adjust.sdk.AdjustEvent;
+import com.adjust.sdk.LogLevel;
+
+import org.mozilla.focus.BuildConfig;
+import org.mozilla.focus.FocusApplication;
+import org.mozilla.focus.telemetry.TelemetryWrapper;
 
 public class AdjustHelper {
-    public static void setupAdjustIfNeeded(Context context) {
-        // BETA: No Adjust - This class has different implementations for all build types.
+    public static void setupAdjustIfNeeded(FocusApplication application) {
+        // NiGHTLY: Enable Adjust - This class has different implementations for all build types.
+
+        //noinspection ConstantConditions
+        if (TextUtils.isEmpty(BuildConfig.ADJUST_TOKEN)) {
+            throw new IllegalStateException("No adjust token defined for release build");
+        }
+
+        if (!TelemetryWrapper.isTelemetryEnabled(application)) {
+            return;
+        }
+
+        final AdjustConfig config = new AdjustConfig(application,
+                BuildConfig.ADJUST_TOKEN,
+                AdjustConfig.ENVIRONMENT_SANDBOX,   //sandbox only
```

Comment:
Looks like we copy the file content from the release one, can we reuse the same file with this parameter different in different flavor?

Reply:
yes. see todo in https://github.com/mozilla-tw/Rocket/pull/2949#issue-232637917

Reply:
fixed